app/assets/javascripts/angular/controller/forms/previewForm.js.erb

```diff
@@ -7,20 +7,36 @@ app.controller('PreviewFormCtrl', ['$scope', '$http', '$auth', '$location', '$ro
   var options = [];
 
   $http.get(url + "/forms/" + $routeParams.id).success(function(response){
+    
     console.log("Form infos",response);
+    
     $scope.form = response;
     $scope.contents = response.contents;
+
+    //Get all the options for every contents
     $scope.contents.forEach(function(content) {
       if (content.options !== null) {
-        console.log('content.options :',content.options)
-        $scope.options = content.options
+
+        console.log('content.options :',content.options);
+        $scope.options = content.options;
+
+        content.options.forEach(function(option) {
```

Comment:
@ctlong @raphaeldieu @yohanology 

If you need to do a lot of these things, use [underscore.js](http://underscorejs.org/)